examples/with-feathersjs/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "with-feathersjs",
+  "description": "",
+  "version": "0.0.0",
+  "homepage": "",
+  "main": "src/",
+  "keywords": [
+    "feathers"
+  ],
+  "license": "MIT",
+  "repository": {},
+  "author": {},
+  "contributors": [],
+  "bugs": {},
+  "engines": {
+    "node": ">= 0.12.0"
+  },
+  "scripts": {
+    "test": "npm run jshint && npm run mocha",
+    "jshint": "jshint src/. test/. --config",
+    ".prestart": "next build",
+    "start": "node src/",
```

Comment:
Thanks @arunoda, I understand and I wondered if the example was even relevant. But for small applications, simple React SSR in Feathers JS and if we want to keep everything in the same place, I think it makes sense.
If Feathers has to be run on a different server, this example is thus irrelevant as no specific setup is needed.